tests/app/dao/test_services_dao.py

```diff
@@ -1099,3 +1071,79 @@ def create_email_sms_letter_template():
     template_two = create_template(service=service, template_name='2', template_type='sms')
     template_three = create_template(service=service, template_name='3', template_type='letter')
     return template_one, template_three, template_two
+
+
+@freeze_time("2019-12-02 12:00:00.000000")
+def test_dao_find_services_sending_to_tv_numbers(notify_db_session, fake_uuid):
+    service_1 = create_service(service_name="Service 1", service_id=fake_uuid)
+    service_3 = create_service(service_name="Service 3", restricted=True)  # restricted is excluded
+    service_4 = create_service(service_name="Service 4", research_mode=True)  # research mode is excluded
+    service_5 = create_service(service_name="Service 5", active=False)  # not active is excluded
+    services = [service_1, service_3, service_4, service_5]
+
+    tv_number = "447700900001"
+    normal_number = "447711900001"
+    normal_number_resembling_tv_number = "447227700900"
+
+    for service in services:
+        template = create_template(service)
+        for x in range(0, 5):
+            create_notification(template, normalised_to=tv_number, status="permanent-failure")
+
+    service_6 = create_service(service_name="Service 6")  # notifications too old are excluded
+    with freeze_time("2019-11-30 15:00:00.000000"):
+        template_6 = create_template(service_6)
+        for x in range(0, 5):
+            create_notification(template_6, normalised_to=tv_number, status="permanent-failure")
+
+    service_2 = create_service(service_name="Service 2")  # below threshold is excluded
+    template_2 = create_template(service_2)
+    create_notification(template_2, normalised_to=tv_number, status="permanent-failure")
```

Comment:
Should we create some `test` notifications?

Reply:
Good idea, I will add those.